features/smoke_tests/handled.feature

```diff
@@ -7,9 +7,12 @@ Scenario: Notify caught Java exception with default configuration
 
     # Exception details
     And the error payload field "events" is an array with 1 elements
+    And the error payload field "events.0.device.runtimeVersions.androidApiLevel" is not null
+    And the error payload field "events.0.device.runtimeVersions.osBuild" is not null
     And the exception "errorClass" equals "java.lang.IllegalStateException"
     And the exception "message" equals "HandledJavaSmokeScenario"
     And the exception "type" equals "android"
+    And the event "context" equals "FooContext"
```

Comment:
It would be good for this assertion to be covered on the handled NDK, unhandled JVM, and unhandled NDK errors too - this appears to be a missing gap from the original tests

Reply:
Done.